app/controllers/assessments_controller.rb

```diff
@@ -607,8 +607,11 @@ def valid_json?(json)
 
   def reload
     @assessment.load_config_file
-  rescue StandardError
+    # rubocop:disable Lint/RescueException
+  rescue Exception => e
```

Comment:
`rescue Exception` should be avoided + best to avoid disabling the linter; try `rescue SyntaxError` instead